sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/plans/LogicalPlanSuite.scala

```diff
@@ -89,4 +90,14 @@ class LogicalPlanSuite extends SparkFunSuite {
     assert(TestBinaryRelation(relation, incrementalRelation).isStreaming === true)
     assert(TestBinaryRelation(incrementalRelation, incrementalRelation).isStreaming)
   }
+
+  test("getAliasedConstraints") {
+    val expressionNum = 100
+    val aggExpression = (1 to expressionNum).map(i => Alias(Count(Literal(1)), s"cnt$i")())
+    val aggPlan = Aggregate(Nil, aggExpression, LocalRelation())
+
+    val expressions = aggPlan.validConstraints
+    // The size of Aliased expression is n * (n - 1) / 2 + n
+    assert( expressions.size === expressionNum * (expressionNum - 1) / 2 + expressionNum)
```

Comment:
Test on the size of `validConstraints` may be fragile. I think we only need to test if the new API of `ExpressionSet` works.